webpack/components/extensions/HostDetails/Cards/ContentViewDetailsCard/ChangeHostCVModal.js

```diff
@@ -0,0 +1,258 @@
+import React, { useState, useCallback } from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage } from 'react-intl';
+import { useDispatch, useSelector } from 'react-redux';
+import { Modal, Button, Select, SelectOption, Alert, Flex } from '@patternfly/react-core';
+import {
+  global_palette_black_600 as pfDescriptionColor,
+} from '@patternfly/react-tokens';
+import { translate as __ } from 'foremanReact/common/I18n';
+import { STATUS } from 'foremanReact/constants';
+import { useAPI } from 'foremanReact/common/hooks/API/APIHooks';
+import { HOST_DETAILS_KEY } from 'foremanReact/components/HostDetails/consts';
+import { selectAPIStatus } from 'foremanReact/redux/API/APISelectors';
+import EnvironmentPaths from '../../../../../scenes/ContentViews/components/EnvironmentPaths/EnvironmentPaths';
+import { ENVIRONMENT_PATHS_KEY } from '../../../../../scenes/ContentViews/components/EnvironmentPaths/EnvironmentPathConstants';
+import api from '../../../../../services/api';
+import getContentViews from '../../../../../scenes/ContentViews/ContentViewsActions';
+import { selectContentViews, selectContentViewStatus } from '../../../../../scenes/ContentViews/ContentViewSelectors';
+import { uniq } from '../../../../../utils/helpers';
+import ContentViewIcon from '../../../../../scenes/ContentViews/components/ContentViewIcon';
+import updateHostContentViewAndEnvironment from './HostContentViewActions';
+import HOST_CV_AND_ENV_KEY from './HostContentViewConstants';
+
+const ENV_PATH_OPTIONS = { key: ENVIRONMENT_PATHS_KEY };
+
+const ContentViewDescription = ({ cv, versionNumber }) => {
+  const descriptionStyle = {
+    fontSize: '12px',
+    fontWeight: 400,
+    color: pfDescriptionColor.value,
+  };
+  if (cv.default) return <span style={descriptionStyle}>{__('Library')}</span>;
+  return (
+    <span style={descriptionStyle}>
+      <FormattedMessage
+        id={`content-view-${cv.id}-version-${cv.latest_version}`}
+        defaultMessage="Version {versionNumber}"
+        values={{ versionNumber }}
+      />
+    </span>
+  );
+};
```

Comment:
You may be wondering, Why not just use the `description` prop of `<SelectOption>`? It's because it makes it impossible to align with the icon as per the mockup, and instead puts it on the far left, under the icon. So I instead did the above, duplicating the styling and importing the same text color from Patternfly, and then using `<Flex>` below to get the layout correct.

Reply:
Can you export this or move it to a component directory so we can add it on the CV UI?